superset/assets/src/chart/ChartRenderer.jsx

```diff
@@ -201,14 +173,40 @@ class ChartRenderer extends React.Component {
 
     const skipChartRendering = isLoading || !!chartAlert || chartStatus === null;
     this.renderStartTime = Logger.getTimestamp();
+
+    if (skipChartRendering) {
```

Comment:
should this be before we set the renderStartTime?

Reply:
yes!